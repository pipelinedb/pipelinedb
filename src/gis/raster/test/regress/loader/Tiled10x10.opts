-t 10x10 -C
