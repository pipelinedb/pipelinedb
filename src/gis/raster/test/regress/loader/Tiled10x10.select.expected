0|1.0000000000|-1.0000000000|10|10|t|f|3|{8BUI,8BUI,8BUI}|{NULL,NULL,NULL}|{f,f,f}|POLYGON((0 -50,0 0,90 0,90 -50,0 -50))
POLYGON((0 0,1 0,1 -1,0 -1,0 0))|255
POLYGON((40 -20,41 -20,41 -21,40 -21,40 -20))|0
POLYGON((80 -40,81 -40,81 -41,80 -41,80 -40))|198
